plugins/importexport/pubmed/PubMedExportPlugin.php

```diff
@@ -191,14 +191,22 @@ public function exportIssues($issueIds, $context, $user)
         $pubmedExportFilters = $filterDao->getObjectsByGroup('article=>pubmed-xml');
         assert(count($pubmedExportFilters) == 1); // Assert only a single serialization filter
         $exportFilter = array_shift($pubmedExportFilters);
-        $submissions = Repo::submission()
-            ->getCollector()
-            ->filterByContextIds([$context->getId()])
-            ->filterByIssueIds($issueIds)
-            ->getMany();
+        $input = [];
+
+        foreach ($issueIds as $issueId) {
+            $sections = Repo::section()->getByIssueId($issueId);
+            foreach ($sections as $section) {
+                $submissionsInSections = [];
+                $submissionsInSections[] = Repo::submission()->getInSections($issueId, $context->getId());
```

Comment:
I think either you can move this line out of/before `foreach ($sections...` or 
you can use 
```
$this->getCollector()
  ->filterByContextIds([$contextId])
  ->filterByIssueIds([$issueId])
  ->filterBySectionIds([$section->getId()])
  ->orderBy(Collector::ORDERBY_SEQUENCE, Collector::ORDER_DIR_ASC)
  ->getMany();
```

Reply:
I noticed that if I don't have `$submissionsInSections = [];` at the beginning to reset the array, then the export will have duplicated articles (because it still has data from the previous issue in the array)

Reply:
maybe you can use `unset($submissionsInSections)` at the end of the loop?